integration_tests/README.md

```diff
@@ -303,3 +303,94 @@ The marks you care about are all in marks.py
 
 For the most part you can ignore this file. It provides the underlying Spark session to operations that need it, but most tests should interact with
 it through `asserts.py`.
+
+## Guidelines for Testing
+
+When support for a new operator is added to `rapids-plugin-4-spark`, it is recommended that
+the following conditions be covered in its corresponding integration tests:
+
+### 1. Cover all supported data types
+Ensure that tests cover all data types supported by the added operation. An exhaustive list of data types supported in 
```

Comment:
Should this explicitly mention negative testing of data types that Spark supports for the operation?  Many times developers will interpret "all supported data types" as testing only the types supported by the GPU operation.  We also want to ensure that any types not supported by the GPU properly fallback to the CPU when executing.

Bonus points for pointing the reader to helper classes or examples that can assist with data type negative testing which is a bit trickier in practice.

Reply:
We do support negative tests.

`assert_gpu_fallback_collect`

Reply:
Yes, I was thinking this doc should mention `assert_gpu_fallback_collect`.  That's probably sufficient for now, devs can go looking for existing use-cases of that method to find examples.

Reply:
Agreed. I've added mention of `assert_gpu_fallback_collect()`.